textbook/11/1/causality.ipynb

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    x = np.random.uniform(0,1,size=100)
</pre><p> </p><p>So, I referred to PEP-8 because its suggestions are useful for anyone programming (in Python and otherwise), and because this stuff is mega-boring (and piddling) to discuss. Anyway, I'm sure there are more interesting things to discuss here. Nonetheless, this is the sort of thing that should be correct for a PR – whether the work is going to be used in a programming library or published in a textbook.</p><p> </p><p>What I see here is truly trivial. For example: spacing between arguments. But any such thing not caught by a contributor will have to be caught by someone else instead. (It's little different from a "typo" in prose.)</p><p> </p><p>This might be instead:</p><p> </p><pre class="ql-syntax" spellcheck="false">x = np.random.uniform(0, 1, size=100)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>